pallets/settlement/src/lib.rs

```diff
@@ -621,7 +618,7 @@ decl_module! {
             T::MaxNumberOfOffChainAssets::get()
         )]
         pub fn reject_instruction(origin, id: InstructionId, portfolio: PortfolioId) -> DispatchResultWithPostInfo {
-            Self::base_reject_instruction(origin, id, portfolio)
+            Self::base_reject_instruction(origin, id, portfolio, None)
         }
```

Comment:
We should be able to use the new weight functions and just pass `None` to them for these old extrinsics.

Reply:
Also we wouldn't need the benchmarks for the old extrinsics, but leave them in for now so we can compare the generated weight functions.